src/js/media-seek-forward-button.js

```diff
@@ -69,13 +53,51 @@ class MediaSeekForwardButton extends MediaChromeButton {
       if (newValue == undefined) {
         this.setAttribute(Attributes.SEEK_OFFSET, DEFAULT_SEEK_OFFSET);
       }
-      updateSeekIconValue(this);
+      updateSeekIconValue(this, 'forward');
       updateAriaLabel(this);
     }
 
     super.attributeChangedCallback(attrName, _oldValue, newValue);
   }
 
+  // Own props
+
+  /**
+   * @type {number} Seek amount in seconds
+   */
+  get seekOffset() {
+    return +this.getAttribute(Attributes.SEEK_OFFSET);
+  }
+
+  set seekOffset(value) {
+    // avoid triggeting a set if no change
+    if (value == this.seekOffset) return;
+    if (value == null) {
+      this.removeAttribute(Attributes.SEEK_OFFSET);
+      return;
+    }
+    this.setAttribute(Attributes.SEEK_OFFSET, `${+value}`);
+  }
+
+  // Props derived from Media UI Attributes
+
+  /**
+   * The current time
+   * @type {number | undefined} In seconds
+   */
+  get mediaCurrentTime() {
+    const attrVal = this.getAttribute(MediaUIAttributes.MEDIA_CURRENT_TIME);
+    return attrVal != null ? +attrVal : undefined;
+  }
+
+  set mediaCurrentTime(time) {
+    if (time == null) {
+      this.removeAttribute(MediaUIAttributes.MEDIA_CURRENT_TIME);
+      return;
+    }
+    this.setAttribute(MediaUIAttributes.MEDIA_CURRENT_TIME, time.toString(10));
```

Comment:
```suggestion
    this.setAttribute(MediaUIAttributes.MEDIA_CURRENT_TIME, `${+time}`);
```

can we also use `${+time}` here to keep that consistent a bit ?

Reply:
I pushed this change up right as you where commenting 😛